src/components/Register.jsx

```diff
@@ -1,12 +1,42 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 
 const Register = () => {
-  const [name, setName] = useState("");
-  const [userName, setUserName] = useState("");
-  const [password, setPassword] = useState("");
+  const nagivate = useNavigate();
+  const [formInputs, setFormInputs] = useState({
+    id: new Date().getTime(),
+    name: "",
+    firstName: "",
+    password: "",
+  });
+
+  let savedProfiles = [formInputs];
+  if (localStorage.getItem("profileData")) {
+    savedProfiles = JSON.parse(localStorage.getItem("profileData"));
+  }
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+
+    setFormInputs((prev) => {
+      return { ...prev, [name]: value };
+    });
+  };
 
   const handleSubmit = (e) => {
     e.preventDefault();
+    const isExists = savedProfiles?.some((profile) => {
```

Comment:
some will return boolean value. we can just check here if names are same or not, if true then we can use alert with this `isExists` variable. Also `isExists` doesn't sound suitable, maybe we can name it `isProfileExists`